src/scripts/gfi-discord-webhook.ts

```diff
@@ -0,0 +1,71 @@
+import * as dotenv from "dotenv"
+
```

Comment:
In the `getTranslationProgress.ts` script we have `import "dotenv/config"`, which points to `node_modules/dotenv/config.d.ts` containing `export {};`... This may be all we need to get the default env vars from the workflow

```suggestion
```